plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -535,9 +538,12 @@ private void addSystemWideContainerSettings(String workspaceId,
 
             volumes = new ArrayList<>(devMachineSystemVolumes.size() + 1);
             volumes.addAll(devMachineSystemVolumes);
-            String projectFolderVolume = format("%s:%s:Z",
+            String projectFolderVolume = format("%s:%s",
                                                 workspaceFolderPathProvider.getPath(workspaceId),
                                                 projectFolderPath);
+            if (volumeOptions != null && !volumeOptions.isEmpty()) {
```

Comment:
you should use guava helper isNullOrEmpty / com.google.common.base.Strings.isNullOrEmpty;